example-generator/src/main/java/org/ehrbase/openehr/sdk/examplegenerator/ExampleGeneratorConfig.java

```diff
@@ -312,13 +320,30 @@ static void handleIsmTransition(IsmTransition value, WebTemplateNode node) {
             Transition transition = transitions.iterator().next();
             State currentState = transition.getTargetState();
 
+            List<WebTemplateInput> cfs = getInputs(node.findChildById("careflow_step").orElseThrow(), t -> true);
```

Comment:
Do we really expect multiple `careflow_step`s?

Reply:
As fae as I know, a web template can configure multiple valid careflow_steps!

Reply:
There is of cause one careflow_step WebTemplateNode in the webtemplate with 0..1 instances in the IsmTransition.
There should be only one input with name "code" in the "careflow_step" node, as with "transition".

It careflow_steps are available, they must constrain the available states before selecting current_state and transition.
Also, the selected careflow_step needs to be constrained by the available states.
Thisway current_state, transition and careflow_step all agree on the state.